_topic_map.yml

```diff
@@ -1285,12 +1285,12 @@ Topics:
   Dir: migrating_4_2_4
```

Comment:
I stumbled across this typo in the above line (it's outside your PR changes so it wouldn't let me comment on line 1284) if you wouldn't mind fixing OpenShift here: `Migrating Openshift Container Platform 4.2 to 4.3`